Styles/alert.css

```diff
@@ -0,0 +1,76 @@
+/* importing external css */
+@import "/Styles/variable.css";
+@import "/Styles/lenght.css";
+@import "/Styles/typography.css";
+
+
+/* --------- */
+
+
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans:wght@600&family=Roboto&display=swap");
+
+
+/* simple alert starts */
+.alert{
+    
+    /* width: 100%; */
+    border: 1px solid transparent;
+    
+   
+    font-family: "Open Sans", sans-serif;
+    font-weight: var(--font-medium);
+    text-align: center;
+    border-radius:10px ;
+    position: relative;
+    
+}
+
+.alert-primary {
+    background-color:var(--primary-bg-alert);
+    
+  }
+  
+  .alert-secondary {
+    background-color: var(--secondary-bg-alert);
+  }
+  
+  .alert-success {
+    background-color: var(--success-bg-alert);
+  }
+  
+  .alert-error {
+    background-color:var(--error-bg-alert);
+  }
+
+
+  
+  /* - alert with icon starts */
+
+  .material-icons{
+    
+     vertical-align: middle;
+     padding: 0 0.3em;
```

Comment:
Please use one particular unit.

Reply:
above three changes are made in the new commit. 
Pls approve if things are looking fine.